application/views/user/template.php

```diff
@@ -75,8 +75,8 @@ function hideURLbar(){ window.scrollTo(0,1); } </script>
 				<!-- Collect the nav links, forms, and other content for toggling -->
 				<div class="collapse navbar-collapse menu--shylock" id="bs-example-navbar-collapse-1">
 				  <ul class="nav navbar-nav menu__list">
-					<li class="menu__item"><a class="menu__link" href="<?php echo base_url().'user/welcome/index'?>">Home <span class="sr-only">(current)</span></a></li>
-					<li class=" menu__item"><a class="menu__link" href="<?php echo base_url().'user/welcome/produk'?>">Produk</a></li>
+					<li class="menu__item"><a class="menu__link" href="<?php echo base_url().'index.php/welcome/index'?>">Home <span class="sr-only">(current)</span></a></li>
+					<li class=" menu__item"><a class="menu__link" href="<?php echo base_url().'user/welcome/produk.php'?>">Produk</a></li>

```

Comment:
ini kayaknya index.php/user/welcome/produk ya

Reply:
betul, yang lainnya juga sama. kecuali about masih error. urang push deui we nya, atau sekalian benerin about?

Reply:
di push dl aja. about klo gak salah error nya beda to cmiiw